account_move_batch_validate/account.py

```diff
@@ -25,12 +25,16 @@
 from openerp.osv import fields, orm
 from openerp.tools.translate import _
 
-from openerp.addons.connector.queue.job import job
-from openerp.addons.connector.session import ConnectorSession
-from openerp.addons.connector.queue.job import OpenERPJobStorage
-
 _logger = logging.getLogger(__name__)
 
+try:
+    from openerp.addons.connector.queue.job import job
+    from openerp.addons.connector.session import ConnectorSession
+    from openerp.addons.connector.queue.job import OpenERPJobStorage
+except ImportError:
+    _logger.debug('Can not `import connector`.')
```

Comment:
https://github.com/OCA/runbot-addons/issues/34

But I think the point is that it is good if the failing import makes runbot red because that indicates a missing dependency that should be fixed.